conversion/ena/submission.py

```diff
@@ -93,10 +101,12 @@ def submission_file(data: Submission) -> Element:
             study: Entity
             for study in data.get_entities('study'):
                 if 'release_date' in study.attributes:
-                    release_dates.add(study.attributes['release_date'])
+                    release_date = date.fromisoformat(study.attributes['release_date'])
+                    if release_date > date.today():
```

Comment:
I am not sure it is right. Why are we doing this conditional check for the release date?
[How to Register a Study](https://ena-docs.readthedocs.io/en/latest/submit/study.html)

Reply:
If the release date is in the past there's no point in telling ENA about the release date.
I'm fairly sure that this is responsible for the hold date set for two years in the future because I had specified a hold date in the past.
I will be able to test this more thoroughly when the ENA test systems are back online.

Reply:
Please read the above link. That's why I inserted it there.
If you won't pass any release date the study will be private.
If you pass one that is in the past it will be public immediately.